tests/Rule/HasLengthHandlerTest.php

```diff
@@ -13,45 +13,28 @@ final class HasLengthHandlerTest extends AbstractRuleValidatorTest
 {
     public function failedValidationProvider(): array
     {
-        $defaultConfig = new HasLength(min: 25);
+        $defaultRule = new HasLength(min: 25);
+        $message = 'This value must be a string.';
+        $greaterThanMaxMessage = 'This value must contain at most {max, number} {max, plural, one{character} ' .
+            'other{characters}}.';
+        $notExactlyMessage = 'This value must contain exactly {exactly, number} {exactly, plural, one{character} ' .
+            'other{characters}}.';
+        $lessThanMinMessage = 'This value must contain at least {min, number} {min, plural, one{character} ' .
+            'other{characters}}.';
```

Comment:
These should be replaced with actual values.

Reply:
Decided to it in a separate PR after merging https://github.com/yiisoft/translator/pull/64.